omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -456,6 +456,7 @@
 default['private_chef']['opscode-erchef']['solr_http_cull_interval'] = '{1, min}'
 default['private_chef']['opscode-erchef']['solr_http_max_age'] = '{70, sec}'
 default['private_chef']['opscode-erchef']['solr_http_max_connection_duration'] = '{70,sec}'
+default['private_chef']['opscode-erchef']['retry_on_conn_closed'] = true
```

Comment:
I know it is still a gross misnomer, but I think perhaps we should prefix this with `solr_` still to match the other options?

Reply:
ha! I did go back and forth in doing that, I also though perhaps search_ would be a better option, but since we are removing solr and related things I though perhaps best to leave it without the "solr_" prefix? I dont like that is is inconsistent either.